commit 0a00bc2019ea54875b7263b1b0340c556379708a
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Thu Oct 21 11:04:12 2010 -0700

    staging: brcm80211: Remove unnecessary header files.
    
    These header files are no longer needed.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/proto/bcmip.h
+++ /dev/null
@@ -1,56 +0,0 @@
-/*
- * Copyright (c) 2010 Broadcom Corporation
- *
- * Permission to use, copy, modify, and/or distribute this software for any
- * purpose with or without fee is hereby granted, provided that the above
- * copyright notice and this permission notice appear in all copies.
- *
- * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
- * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
- * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
- * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
- * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
- * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
- * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
- */
-
-#ifndef _bcmip_h_
-#define _bcmip_h_
-
-#include <packed_section_start.h>
-
-#define IP_VER_OFFSET		0x0
-#define IP_VER_MASK		0xf0
-#define IP_VER_SHIFT		4
-#define IP_VER_4		4
-#define IP_VER_6		6
-
-#define IP_VER(ip_body) \
-	((((u8 *)(ip_body))[IP_VER_OFFSET] & IP_VER_MASK) >> IP_VER_SHIFT)
-
-#define IPV4_TOS_OFFSET		1
-
-#define IPV4_ADDR_LEN		4
-
-#define	IPV4_TOS(ipv4_body)	(((u8 *)(ipv4_body))[IPV4_TOS_OFFSET])
-
-#define	IPV4_TOS_PREC_MASK	0xe0
-#define	IPV4_TOS_PREC_SHIFT	5
-
-BWL_PRE_PACKED_STRUCT struct ipv4_addr {
-	u8 addr[IPV4_ADDR_LEN];
-} BWL_POST_PACKED_STRUCT;
-
-#define IPV6_TRAFFIC_CLASS(ipv6_body) \
-	(((((u8 *)(ipv6_body))[0] & 0x0f) << 4) | \
-	 ((((u8 *)(ipv6_body))[1] & 0xf0) >> 4))
-
-#ifndef IP_TOS
-#define IP_TOS(ip_body) \
-	(IP_VER(ip_body) == IP_VER_4 ? IPV4_TOS(ip_body) : \
-	 IP_VER(ip_body) == IP_VER_6 ? IPV6_TRAFFIC_CLASS(ip_body) : 0)
-#endif
-
-#include <packed_section_end.h>
-
-#endif				/* _bcmip_h_ */
--- a/drivers/staging/brcm80211/include/proto/vlan.h
+++ /dev/null
@@ -1,45 +0,0 @@
-/*
- * Copyright (c) 2010 Broadcom Corporation
- *
- * Permission to use, copy, modify, and/or distribute this software for any
- * purpose with or without fee is hereby granted, provided that the above
- * copyright notice and this permission notice appear in all copies.
- *
- * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
- * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
- * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
- * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
- * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
- * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
- * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
- */
-
-#ifndef _vlan_h_
-#define _vlan_h_
-
-#include <packed_section_start.h>
-
-#define VLAN_VID_MASK		0xfff
-#define	VLAN_CFI_SHIFT		12
-#define VLAN_PRI_SHIFT		13
-
-#define VLAN_PRI_MASK		7
-
-#define	VLAN_TAG_LEN		4
-#define	VLAN_TAG_OFFSET		(2 * ETHER_ADDR_LEN)
-
-#define VLAN_TPID		0x8100
-
-struct ethervlan_header {
-	u8 ether_dhost[ETHER_ADDR_LEN];
-	u8 ether_shost[ETHER_ADDR_LEN];
-	u16 vlan_type;
-	u16 vlan_tag;
-	u16 ether_type;
-};
-
-#define	ETHERVLAN_HDR_LEN	(ETHER_HDR_LEN + VLAN_TAG_LEN)
-
-#include <packed_section_end.h>
-
-#endif				/* _vlan_h_ */
